*** ./smali/android/util/DisplayMetrics.smali	2015-09-29 11:58:08.688336913 -0700
--- /home/mrx/rom/one/temp/new_smali/framework.jar.out/./smali/android/util/DisplayMetrics.smali	2015-09-29 11:58:46.112338004 -0700
***************
*** 21,26 ****
--- 21,28 ----
  
  .field public static final DENSITY_MEDIUM:I = 0xa0
  
+ .field public static final DENSITY_NXHGITH:I
+ 
  .field public static final DENSITY_TV:I = 0xd5
  
  .field public static final DENSITY_XHIGH:I = 0x140
***************
*** 65,70 ****
--- 67,78 ----
      .locals 1
  
      .prologue
+     invoke-static {}, Landroid/util/MiuiDisplayMetrics;->getNxhdpiDensity()I
+ 
+     move-result v0
+ 
+     sput v0, Landroid/util/DisplayMetrics;->DENSITY_NXHGITH:I
+ 
      invoke-static {}, Landroid/util/DisplayMetrics;->getDeviceDensity()I
  
      move-result v0
